hdf5-rs/src/hl/space.rs

```diff
@@ -49,6 +54,69 @@ impl Deref for Dataspace {
 }
 
 impl Dataspace {
+
+    /// Create a simple dataspace covering the given `shape`
+    pub fn create_simple(shape: &[Ix]) -> Result<Dataspace> {
```

Comment:
How is this different from `Dataspace::try_new()?` (aside from a better name perhaps) Or should it at least reuse that?

(dataspaces/shapes/dims will likely be heavily refactored in the future anyway to integrate better with ndarray)

Reply:
sorry, missed try_new. Got rid of this method.